GitterIntroAkkaAkka.md

```diff
@@ -1,7 +1,7 @@
 # The akka/akka Gitter Channel
 
-This channel is available for all Akka enthusiasts—newbies as well as gurus—for the exchange of knowledge and the coordination of efforts around Akka; it is a community effort and resource and not backed by Typesafe. For more structured discussions please refer to [the akka-user mailing list](https://groups.google.com/forum/#!forum/akka-user).
+This channel is available for all Akka enthusiasts—newbies as well as gurus—for the exchange of knowledge and the coordination of efforts around Akka; it is a community effort and resource and not backed by Lightbend. For more structured discussions please refer to [the Akka discussion forum](https://discuss.akka.io).
 
-Instead of a long Code of Conduct we rely upon common sense: be kind and respectful to those who are already here and to those who come after you, harassment of any kind will not be tolerated; in case of trouble contact [akka.official](mailto:akka.official@gmail.com).
+The [Lightbend Community Code of Conduct](https://www.lightbend.com/conduct) applies to all Akka Gitter channels, along with all other Akka community resources.
```

Comment:
If you still consider my input, I’d like to speak out against this change. I might well have missed it, but I am not aware of a discussion on this topic, so it is appropriate to discuss it now on akka-meta.

Making this change will make it impossible for me to contribute in any form. You may have noticed that I do not partake in Scala channels anymore since the introduction of the Code of Conduct that has since become the Lightbend Community Code of Conduct. The reason is that I take any rule imposed in a given setting seriously, and I actually do read them. As such, I am sure that even this very comment here constitutes a violation of said code, since the term “micro-aggression” is ill-defined and the code makes it clear that it lies in the eye of the beholder whether behavior constitutes harassment or not. If I cannot be reasonably sure that any given post I make will pass the code, I cannot engage at all. This is the opposite of an inclusive community.

If we wish to officially include a code of conduct for Akka, then I’d be willing to help formulate one that can reasonably be applied by everyone before sending a message.

Reply:
@rkuhn let me clarify that the Akka team, Lightbend, and I all value your input. I don’t consider your comment a violation of the code of conduct—it explicitly encourages respectful and constructive disagreement. If this isn’t clear, then we should try to clarify it. 
 
A code of conduct was originally added to Akka two years ago (https://github.com/akka/akka/pull/23673). The document changed in this pull request was apparently forgotten, and had missed this and other changes over the past few years. What has changed recently is the standardization across several projects on the code of conduct derived from the Scala Code of Conduct. One motivation for this was to make it less ambiguous what constitutes acceptable and unacceptable behavior, and how violations will be handled and by whom. Of course, some interpretation and moderator discretion is required, and it’s impossible to be fully comprehensive, but I believe the current language is more precise than what it replaces, particularly in this document. 
 
A second motivation is to align other projects in the Lightbend ecosystem with the Scala Code of Conduct, in order to reduce the difference between the codes used by closely-related projects. While the Scala Code of Conduct is arguably imperfect, I believe the communities will be better served by unifying and improving the shared code. 
 
Since this pull request isn’t introducing the change to the code of conduct—only updating the Gitter reference to reflect it—I think it would be better to redirect this discussion elsewhere if you’d like to continue it. You can reach me directly at tim.moore@lightbend.com.

Reply:
Thanks for hearing me! First, we should agree on the nature of this change: this PR does change what visitors of the gitter channels see, and as such the statement in your last paragraph is false. Second, a standardization across multiple projects would only be a trivial matter (i.e. not worthy of proper discussion) if the differences between them were negligible to begin with, which is not the case here. Thank you for pointing out the original addition, which is very different in tone and scope from the change that was brought about — without discussion in akka-meta — in akka/akka#27600 three weeks ago. So, before we go into the details, I would like to confirm that this is indeed a change that must be discussed by the Akka project. We can do it in a different issue, but I think it belongs into the akka-meta repository, where it currently is.